tests/testthat/test-bold_italicise_group_labels.R

```diff
@@ -0,0 +1,54 @@
+library(gtsummary)
+library(dplyr)
+
```

Comment:
I don't know if it is valid/necessary to call library() within tests files

Reply:
Hi @ddsjoberg I've pushed the unit tests, please take a look at it and tell me if it is what you expected.